src/documents/jqUnit.md

```diff
@@ -0,0 +1,305 @@
+---
+title: jqUnit
+layout: default
+category: Infusion
+---
+
+Bundled with infusion is ***jqUnit***, a library modelled on the [xUnit](https://en.wikipedia.org/wiki/XUnit) API style which wraps base functionality provided by the
```

Comment:
The "which" clause here is ambiguous. I suggest leaving out the "modeled on the xunit api style" and changing "which" to "that."